src/platform/packages/private/kbn-validate-oas/oas_error_baseline.json

```diff
@@ -1,4 +1,4 @@
 {
-  "./oas_docs/output/kibana.yaml": 312,
-  "./oas_docs/output/kibana.serverless.yaml": 294
+  "./oas_docs/output/kibana.yaml": 322,
+  "./oas_docs/output/kibana.serverless.yaml": 305
```

Comment:
Talked on Slack. The errors have a reason, and are pending on https://github.com/elastic/kibana/issues/198680